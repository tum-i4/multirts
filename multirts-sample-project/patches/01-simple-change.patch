diff --git a/multirts-sample-project/foo/src/main/java/a/b/c/foo/Foo.java b/multirts-sample-project/foo/src/main/java/a/b/c/foo/Foo.java
index 5f91f50..d09b01f 100644
--- a/multirts-sample-project/foo/src/main/java/a/b/c/foo/Foo.java
+++ b/multirts-sample-project/foo/src/main/java/a/b/c/foo/Foo.java
@@ -2,6 +2,6 @@ package a.b.c.foo;
 
 public class Foo {
     public static boolean foo() {
-        return true;
+        return false;
     }
 }
