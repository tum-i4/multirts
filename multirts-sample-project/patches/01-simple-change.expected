module-selection:
    modules:
        foo/pom.xml

test-selection:
    tests:
        a.b.c.foo.FooTest
        a.b.c.bar.FooTestCase

    modules:
        bar/pom.xml
        foo/pom.xml